config/setup/errors.go

```diff
@@ -71,6 +81,16 @@ func errorsParse(c *Controller) (*errors.ErrorHandler, error) {
 
 			if what == "log" {
 				handler.LogFile = where
+				if c.NextArg() {
+					if c.Val() == "{" {
+						c.IncrNest()
```

Comment:
I think you did a very good job here of handling the unusual, nested block. I can see what you mean about how it's not pretty. For now I'll take it and if another nested block situation comes up, I'll put some more effort into improving on this. It's a great start!

Also the tests are great!